scripts/proposals/BOLD/generate.bash

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# OfficeHoursAction: TODO
+
+# arb1:
+# RollupUpgradeSecondaryAction: 0x2A3a4BDbC7c00d5115d297d83A31358B317d4740
+# - new impl at 0x5c93BAB9Ff2Fa3884b643bd8545C625De0633517
+# BoldUpgradeAction: 0xf795ec38701234664f69Dbd761Ee9c511F25ac1D
+
+# nova:
+# RollupUpgradeSecondaryAction: 0x8E1c1555b2Fe22870e7a0A454789b0c92e494ADC
+# - new impl at 0x5c93BAB9Ff2Fa3884b643bd8545C625De0633517
+# BoldUpgradeAction: 0xd25B258B55765c9fb5567eCABB6114b03b0f78b5
+# SetValidatorsAction: 0x2f845d909058200e4E56855C2735975a004a4922
+
+yarn gen:proposalData \
+    --govChainProviderRPC https://arb1.arbitrum.io/rpc \
+    --actionChainIds 1 1 1 1 1 1 \
+    --actionAddresses \
+        0x000000000000000000000000000000000000dead \
+        0x2A3a4BDbC7c00d5115d297d83A31358B317d4740 \
+        0xf795ec38701234664f69Dbd761Ee9c511F25ac1D \
+        0x8E1c1555b2Fe22870e7a0A454789b0c92e494ADC \
+        0xd25B258B55765c9fb5567eCABB6114b03b0f78b5 \
+        0x2f845d909058200e4E56855C2735975a004a4922 \
+    --actionDatas \
```

Comment:
We should also fix this bug, there are some error text that used `actionDatas` instead of `upgradeDatas`, and we should throw when any unexpected flags are provided.
```suggestion
    --upgradeValues 0 0 0 0 0 \
    --predecessor 0x0000000000000000000000000000000000000000000000000000000000000000 \
    --upgradeDatas \
```

Reply:
fixed in https://github.com/ArbitrumFoundation/governance/pull/326/files